packages/api/methods.ts

```diff
@@ -208,3 +208,11 @@ export function updateRule(rule) {
 export function deleteRule(id) {
   return send('api/rule-delete', { id });
 }
+
+export function holdForNextMonth(month, amount) {
+  return send('api/hold-for-next-month', { month, amount });
+}
+
+export function resetHold(month) {
+  return send('api/reset-hold', { month });
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
export function resetBudgetHold(month) {
  return send('api/reset-budget-hold', { month });
}
```